staging/src/k8s.io/client-go/Godeps/Godeps.json

```diff
@@ -16,19 +16,19 @@
 		},
 		{
 			"ImportPath": "github.com/Azure/go-autorest/autorest",
-			"Rev": "58f6f26e200fa5dfb40c9cd1c83f3e2c860d779d"
+			"Rev": "48a8154950f853ae1f2a9c409e0eb1fb71119bc8"
```

Comment:
It would probably be best to split the vendoring of a new version of the dependency into a separate PR.

Reply:
Same for each new/updated dependency.  A separate PR for each one please.  It makes them more visible and easier to review and roll back if necessary,.

Reply:
I tried to piggy back off a recent commit that updated go-autorest and only add github.com/azure/azure-sdk-for-go/arm/dns, but apparently following the steps at https://github.com/kubernetes/community/blob/master/contributors/devel/godep.md seems to update go-autorest and azure-sdk-for-go. 

Would you still like me to break those out?